packages/calypso-build/README.md

```diff
@@ -127,6 +127,6 @@ Use the provided Jest configuration via a preset. In your `jest.config.js` set t
 
```

Comment:
Since `@automattic/calypso-jest` isn't a dependency of `@automattic/calypso-build`, should we just link to the other package here? (Or, should calypso-jest be a dep of calypso-build?)

Reply:
I've moved this section to a README.md in `calypso-jest`.